firstpartydata/first_party_data.go

```diff
@@ -0,0 +1,376 @@
+package firstpartydata
+
+import (
+	"fmt"
+	"github.com/evanphx/json-patch"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/util/jsonutil"
+)
+
+const (
+	siteKey = "site"
+	appKey  = "app"
+	userKey = "user"
+	dataKey = "data"
+
+	userDataKey        = "userData"
+	appContentDataKey  = "appContentData"
+	siteContentDataKey = "siteContentData"
+)
+
+func ExtractGlobalFPDData(req *openrtb_ext.RequestWrapper) (map[string][]byte, error) {
+	//If {site,app,user}.ext.data exists, collect it and remove {site,app,user}.ext.data
```

Comment:
Please reword this comment and promote it directly to `ExtractGlobalFPDData`. ie:
```
// ExtractGlobalFPDData collects ....
func ExtractGlobalFPDData(req *openrtb_ext.RequestWrapper) (map[string][]byte, error) {
```

Reply:
Moved